src/icp_prototype_backend/src/lib.rs

```diff
@@ -25,13 +25,14 @@ use memory::{
     TRANSACTIONS,
 };
 use types::{
-    CanisterApiManager, CanisterApiManagerTrait, IcCdkSpawnManager, IcCdkSpawnManagerTrait,
+    Network, CanisterApiManager, CanisterApiManagerTrait, IcCdkSpawnManager, IcCdkSpawnManagerTrait,
     InterCanisterCallManager, InterCanisterCallManagerTrait, Operation, QueryBlocksRequest,
     QueryBlocksResponse, StoredPrincipal, StoredTransactions, SweepStatus, TimerManager,
     TimerManagerTrait, Timestamp,
 };
 
 thread_local! {
+    static NETWORK: RefCell<Network> = RefCell::new(Network::Local);
```

Comment:
Add this in `memory.rs` to make it part of the permanent memory block.

Reply:
added NETWORK as CONNECTED_NETWORK in memory.rs